nautobot/dcim/factory.py

```diff
@@ -5,7 +5,11 @@
 from django.db.models import Q
 import factory
 from faker import Faker
-import pytz
+
+try:
+    import zoneinfo
+except ImportError:  # python 3.8
+    from backports import zoneinfo
```

Comment:
Do we need an explicit dependency on `backports.zoneinfo` in our `pyproject.toml`? We get this through another dependency now.

Reply:
Looks like this is coming from Django. I think that's good enough.

Reply:
It comes in as a transitive dependency from Django itself, so I think that's good enough:

```toml
[[package]]
name = "django"
version = "4.2.11"
description = "A high-level Python web framework that encourages rapid development and clean, pragmatic design."
optional = false
python-versions = ">=3.8"
files = [
    {file = "Django-4.2.11-py3-none-any.whl", hash = "sha256:ddc24a0a8280a0430baa37aff11f28574720af05888c62b7cfe71d219f4599d3"},
    {file = "Django-4.2.11.tar.gz", hash = "sha256:6e6ff3db2d8dd0c986b4eec8554c8e4f919b5c1ff62a5b4390c17aff2ed6e5c4"},
]

[package.dependencies]
asgiref = ">=3.6.0,<4"
"backports.zoneinfo" = {version = "*", markers = "python_version < \"3.9\""}
sqlparse = ">=0.3.1"
tzdata = {version = "*", markers = "sys_platform == \"win32\""}
```